src/usr/local/www/system_advanced_admin.php

```diff
@@ -556,6 +580,7 @@
 
 	hideInput('ssl-certref', $('input[name=webguiproto]:checked').val() == 'http');
 	hideCheckbox('webgui-hsts', $('input[name=webguiproto]:checked').val() == 'http');
+	hideCheckbx('ocsp-staple', $(cert_get_ocspstaple(lookup_cert($config['system']['webgui']['ssl-certref'])) == true);
```

Comment:
One more thing I noticed: This is JavaScript here, not PHP, so it should probably be something closer to:

    hideCheckbx('ocsp-staple', <?= cert_get_ocspstaple(lookup_cert($config['system']['webgui']['ssl-certref'])) ? "true" : "false"; ?>);

Reply:
and I spelled hideCheckbox wrong!

Reply:
Forgive ignorance, but is it:

`<?= cert_get_ocspstaple(lookup_cert($config['system']['webgui']['ssl-certref'])) ? "true" : "false"; ?>`

or 

`<?php cert_get_ocspstaple(lookup_cert($config['system']['webgui']['ssl-certref'])) ? "true" : "false"; ?>`

Reply:
You can use either method so long as you handle it properly.
<?= $foo ?> performs an implied echo/print of the returned value, which makes cases like this easier.
<?php ... ?> requires you to do your own echo, so as written in your example it would output nothing, but if you put `echo ` before the function call it would have the same net effect.

Reply:
ok. i pushed the `<?=` version already.

Reply:
and for my own enlightenment, the other option would be:

```<?php echo cert_get_ocspstaple(lookup_cert($config['system']['webgui']['ssl-certref'])) ? "true" : "false"; ?>```

or would it need to be

```<?php cert_get_ocspstaple(lookup_cert($config['system']['webgui']['ssl-certref'])) ? echo "true" : echo "false"; ?>```

Reply:
Either of those should have the same effect in the end.